templates/create-daml-app-test-resources/index.test.ts

```diff
@@ -25,43 +26,36 @@ let uiProc: ChildProcess | undefined = undefined;
 // Chrome browser that we run in headless mode
 let browser: Browser | undefined = undefined;
 
-let publicUser: string = '';
 let publicParty: string = '';
 
-// Function to generate unique party names for us.
-let nextPartyId = 1;
-const getParty = async () : [string, string] => {
-  // TODO For now we use grpcurl to allocate parties and users.
-  // Once the JSON API exposes party & user management we can switch to that.
-  const grpcurlPartyArgs = [
-    "-plaintext",
-    "localhost:6865",
-    "com.daml.ledger.api.v1.admin.PartyManagementService/AllocateParty",
-  ];
-  const allocResult = spawnSync('grpcurl', grpcurlPartyArgs, {"encoding": "utf8"});
-  const parsedResult = JSON.parse(allocResult.stdout);
-  const user = `u${nextPartyId}`;
-  const party = parsedResult.partyDetails.party;
-  const createUser = {
-    "user": {
-      "id": user,
-      "primary_party": party,
-    },
-    "rights": [{"can_act_as": {"party": party}}, {"can_read_as": {"party": publicParty}}]
-  };
-  const grpcurlUserArgs = [
-    "-plaintext",
-    "-d",
-    JSON.stringify(createUser),
-    "localhost:6865",
-    "com.daml.ledger.api.v1.admin.UserManagementService/CreateUser",
-  ];
-  const result = spawnSync('grpcurl', grpcurlUserArgs, {"encoding": "utf8"});
-  nextPartyId++;
-  return [user, party];
+const getPublic = async () : Promise<string> => {
+  const token = authConfig.makeToken('admin');
```

Comment:
where does `admin` come from? afaik the default user is called `participant_admin` not `admin.`